### PR TITLE
Added a "remove damage flash" function

### PR DESCRIPTION
**Pull request by vincentlau36**
An accessibility change to increase ease of play for players with sensitivities to flashing screens.

### CODE REVIEW

File: ValheimPlus/Hud.cs [hunk 1]
```diff
@@ -62,7 +62,16 @@ private static void Postfix(Skills __instance, Skills.SkillType skillType, float
...
 			}
+		}
 
+		[HarmonyPatch(typeof(Hud), "DamageFlash")]
+		public static class RemoveDamageFlash
+		{
+			private static void Prefix(Hud __instance)
+			{
+				if (Configuration.Current.Hud.IsEnabled && Configuration.Current.Hud.removeDamageFlash)
+					__instance.m_damageScreen.enabled = false;
```
Comment (radmint) [inline ValheimPlus/Hud.cs:73]: Thanks for this, where is the `m_damageScreen.enabled` bool coming from?

File: ValheimPlus/Hud.cs [hunk 2]
```diff
@@ -62,7 +62,16 @@ private static void Postfix(Skills __instance, Skills.SkillType skillType, float
 					__instance.m_player.Message(MessageHud.MessageType.TopLeft, skill.m_info.m_skill + " [" + Helper.tFloat(skill.m_accumulator, 2) + "/" + Helper.tFloat(skill.GetNextLevelRequirement(), 2) + "] (" + Helper.tFloat(percent, 0) + "%)", 0, skill.m_info.m_icon);
 				}
 			}
+		}
 
+		[HarmonyPatch(typeof(Hud), "DamageFlash")]
+		public static class RemoveDamageFlash
+		{
+			private static void Prefix(Hud __instance)
```
Comment (Mixone-FinallyHere) [inline ValheimPlus/Hud.cs:70]: Needs to be PostFix or have a return false added

File: ValheimPlus/Hud.cs (hunk 1 above)
Comment (Mixone-FinallyHere) [inline ValheimPlus/Hud.cs:73]: .enabled is the equivalent of .SetActive(bool newvalue)

the normal code, launches .SetActive(true) rendering this .enabled moot

**Review by Mixone-FinallyHere — CHANGES_REQUESTED** [review]
Apart from the small point of the postfixc instead of prefix, or at least a return false looks goood

File: ValheimPlus/Hud.cs (hunk 2 above)
Comment (vincentlau36) [reply ValheimPlus/Hud.cs]: I've made the change now to use Postfix instead. Please check [this commit](https://github.com/valheimPlus/ValheimPlus/pull/141/commits/b9849cdcbfd6305d8c9bacf8978c6bd04628278a)

File: ValheimPlus/Hud.cs (hunk 1 above)
Comment (vincentlau36) [reply ValheimPlus/Hud.cs]: I've changed it to postfix and updated the code to use the SetActive method to set said flag to false. Thank you for the suggestions! Change is in [this commit](https://github.com/valheimPlus/ValheimPlus/pull/141/commits/3936b745cb6d2f232a1ee86ac722051e19b1d0f0)

**Review by radmint — APPROVED** [review]

**Review by Mixone-FinallyHere — APPROVED** [review]
Looks good to me

**Comment by radmint** [comment]
@vincentlau36 just a merge conflict to resolve before this can merge and then it'll be good to go! Thank you!

**Comment by radmint** [comment]
Added "pending close" to close if no updates are provided here

**Comment by vincentlau36** [comment]
Let me know if there's anything I can do to help. Otherwise thanks for the hard work!

**Comment by radmint** [comment]
@vincentlau36 please address the merge conflict. This can't be merged until you do.

**Comment by vincentlau36** [comment]
> @vincentlau36 please address the merge conflict. This can't be merged until you do.

image

@radmint I'm not sure I can. Github desktop isn't giving me anything to review and I got the message above when trying to do so from the web client. Any ideas on how I can address this?

**Comment by radmint** [comment]
Here you go @vincentlau36 I've resolved the merge conflicts for you in this PR https://github.com/valheimPlus/ValheimPlus/pull/230 I hope that's okay. I retained your commits so you will be able to listed as a Contributor still

**Comment by vincentlau36** [comment]
@radmint That's great, thank you!

**Comment by radmint** [comment]
You're welcome! I'm going to close this so keep an eye on the other PR to know when it goes in!

**Pull request closed without merging**